### PR TITLE
refactor(LedgerScan): code

### PR DESCRIPTION
**Pull request by Crazy-MIXER**
- Change the imports to absolute
- Replace navigation hook
- Move text to i18n
- Change the colors on `Color`

### CODE REVIEW

**Review by gHashTag — DISMISSED** [review]

File: src/components/ledger-scan/ledger-scan-row.tsx [hunk 1]
```diff
@@ -23,7 +23,6 @@ export const LedgerScanRow = ({item, onPress}: LedgerScanRowProps) => {
 
 const page = StyleSheet.create({
```
[the remark this answers is not in the document]
Comment (Crazy-MIXER) [reply src/components/ledger-scan/ledger-scan-row.tsx]: done

[the remark this answers is not in the document]
Comment (Crazy-MIXER) [reply src/components/ledger-scan/ledger-scan-row.tsx]: Done
image

File: src/components/ledger-scan/ledger-scan-row.tsx [hunk 2]
```diff
@@ -14,16 +14,15 @@ export type LedgerScanRowProps = {
...
+      <Text color={Color.bg3} style={style.container} t11>
         {item.name}
       </Text>
     </TouchableOpacity>
   );
 };
 
-const page = StyleSheet.create({
+const style = StyleSheet.create({
   container: {
```
Comment (gHashTag) [inline src/components/ledger-scan/ledger-scan-row.tsx:25]:
```suggestion
  textName: {
```

Comment (Crazy-MIXER) [reply src/components/ledger-scan/ledger-scan-row.tsx]: done

**Review by gHashTag — APPROVED** [review]

**Comment by Crazy-MIXER** [comment]
need fix i18n.ts in the branch `main`

**Review by vivalaakam — APPROVED** [review]

**Pull request merged**